src/codewhisperer/service/inlineCompletionService.ts

```diff
@@ -89,7 +89,7 @@ class CWInlineCompletionItemProvider implements vscode.InlineCompletionItemProvi
 
     truncateOverlapWithRightContext(document: vscode.TextDocument, suggestion: string): string {
         let rightContextRange: vscode.Range | undefined = undefined
-        const pos = RecommendationHandler.instance.startPos
+        const pos = vscode.window.activeTextEditor?.selection.active || RecommendationHandler.instance.startPos
```

Comment:
Does the `suggestion: string` contain typeahead?

Reply:
yes in this case, it's usually when user type a html tag like `<button>` or quotes and the IDE filled in the closing tag or quotes